### ISSUE TITLE
Questions about the recalls of  Scene Graph Classification

### ISSUE THREAD

**Issue by Sundrops** [rowanz/neural-motifs#25]
Thanks for your excellent work.
When I evaluated the recalls of Scene Graph Classification using your pretrained sgcls model(vgrel-motifnet-sgcls.tar), I got higher results than your paper's.
```
======================sgcls============================
R@100: 0.417541  vs 36.5 in your paper
R@50: 0.410185 vs 35.8 in your paper
R@20: 0.384014 vs 32.9 in your paper 
```
What's the reason? the better detector?

**Comment by rowanz** [comment]
hi there! I'm really not sure as this checkpoint is the one from the paper. is it possible you're evaluating on the training set?

**Comment by Sundrops** [comment]
It's evaluated on the val set.
And the recalls of predcls and sgdet are here.
```
======================predcls============================
R@100: 0.677808   vs 67.1 in your paper
R@50: 0.663312 vs 65.2 in your paper 
R@20: 0.611537 vs 58.5 in your paper
======================sgdet============================
R@100: 0.302774  vs 30.3 in your paper
R@50: 0.273415 vs 27.2 in your paper
R@20: 0.219299 vs 21.4 in your paper
```

**Comment by rowanz** [comment]
My reported numbers are on the test set, so maybe because the val set is
small that explains the fluctuations?

**Comment by Sundrops** [comment]
oh, thanks a lot.

**Comment by jnhwkim** [comment]
It's consistent with my results. The `val` results are significantly better than `test` results for `sgcls`.
